src/fr/articles/landmarks-aria/index.njk

```diff
@@ -1,137 +1,235 @@
+  <p>Note : si une page <abbr>HTML</abbr> contient un <code>iframe</code>, celui-ci peut également contenir une balise <code>main</code> (ou un attribut <code>role="main"</code>) car c'est une page autonome.</li></p>
+  <p>Pour plus détails, voir <a href="https://adrianroselli.com/2025/04/mainlining-mains.html" lang="en" hreflang="en">Mainlining Mains</a>.</p>
+
+  <h4 id="form">form</h4>
+  <p>La balise <code>form</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="form"</code>) est à utiliser pour identifier un formulaire. Ses enfants seront des balises d'éléments de formulaires comme les balises <code>input</code>, <code>select</code>, <code>textarea</code> ou <code>button</code>.</p>
+  <p>Il peut y avoir plusieurs balises <code>form</code> (ou attributs <code>role="form"</code>) dans une même page. Dans ce cas, y ajouter un nom accessible unique (voir <a href="#bonnes_pratiques">Bonnes pratiques</a>).</p>
+  
+  <h4 id="section">section</h4>
+  <p>La balise <code>section</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="region"</code>) permet de regrouper du contenu (exemples : chapitres d'un livre, grandes parties d'une page d'accueil, ...) thématiquement.</p>
+  <p>Plusieurs balises <code>section</code> (ou attributs <code>role="region"</code>) peuvent être présentes dans la page. Un nom accessible unique est requis pour chaque occurrence (voir <a href="#bonnes_pratiques">Bonnes pratiques</a>).</p>
+  
+  <h4 id="aside">aside</h4>
+  <p>La balise <code>aside</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="complementary"</code>) est à utiliser pour identifier du contenu annexe au contenu principal de la page (information complémentaire, publicité…).</p>
+  <p>Cet élément est à placer comme enfant direct de la base <code>body</code>.</p>
+  <p> Il peut y avoir plusieurs balises <code>aside</code> (ou attributs <code>role="complementary"</code>) dans une même page. Dans ce cas, y ajouter systématiquement un nom accessible unique (voir <a href="#bonnes_pratiques">Bonnes pratiques</a>).</p>
+
+  <h4 id="footer-tag">footer</h4>
+  <p>La balise <code>footer</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="contentinfo"</code>) permet d’identifier le pied de page. Cela est vrai uniquement si cet élément est placé dans une balise <code>body</code>,
+      soit comme enfant direct, soit dans une balise HTML autre que <code>article</code>, <code>aside</code>, <code>main</code>, <code>nav</code> et <code>section</code>.</p>
+  <p>De manière générale et pour simplifier, il est conseillé d'utiliser la balise <code>footer</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="contentinfo"</code>) une seule fois par page.</p>
+  <p>Note : si une page <abbr>HTML</abbr> contient un <code>iframe</code>, celui-ci peut également contenir une balise <code>main</code> (ou un attribut <code>role="main"</code>) car c'est une page autonome.</li></p>
```

Comment:
```suggestion
  <p>Note : si une page <abbr>HTML</abbr> contient un <code>iframe</code>, celui-ci peut également contenir une balise <code>main</code> (ou un attribut <code>role="main"</code>) car c'est une page autonome.</p>
```

Reply:
Not sure why there is a `</li>` here? Couly check it please?

Reply:
You're right, inappropriate autocompletion. Corrected.